lua/nvim-tree/colors.lua

```diff
@@ -52,6 +53,7 @@ local function get_links()
   return {
     FolderName = 'Directory',
     EmptyFolderName = 'Directory',
+    OpenedFolderName = 'ErrorMsg',
```

Comment:
yeah don't link error to opened folder too, this is disturbing. by default you should link to Directory just like the other two